src/common/transformations/src/transformations/common_optimizations/mark_precision_sensitive_subgraphs.cpp

```diff
@@ -32,10 +32,10 @@ bool ov::pass::MarkPrecisionSensitiveSubgraphs::run_on_model(const std::shared_p
         visited.insert(r.get());
     }
 
-    auto markup_func = [](Node* node) {
-        if (ov::is_type<ov::opset8::Constant>(node)) {
-            ov::disable_fp16_compression(node->shared_from_this());
-        }
+    auto markup_func = [this](Node* node) {
```

Comment:
I don't think you need `this` here.

Reply:
without this we cannot access to `m_mark_only_consts` which we need to know we should mark the whole subgraph or only constants